.github/workflows/release.yml.template

```diff
@@ -70,14 +75,30 @@ jobs:
       -
         name: Publish Wasm policy artifact to OCI registry with the 'latest' tag
         if: {{ "${{ startsWith(github.ref, 'refs/heads/') }}" }}
+        env:
+          COSIGN_EXPERIMENTAL: 1
         run: |
-          {{ "./kwctl push policy-annotated.wasm ${{ env.OCI_TARGET }}:latest" }}
+          set -ex
+          echo Pushing policy to OCI container registry
+          {{ "IMMUTABLE_REF=$(kwctl push -o json policy-annotated.wasm ${{ env.OCI_TARGET }}:latest | jq -r .immutable_ref)" }}
+
+          echo Keyless signing of policy using cosign
+          {{ "cosign sign ${IMMUTABLE_REF}" }}
+
       -
         name: Publish Wasm policy artifact to OCI registry with the version tag and 'latest'
         if: {{ "${{ startsWith(github.ref, 'refs/tags/') }}" }}
+        env:
+          COSIGN_EXPERIMENTAL: 1
         run: |
+          set -ex
           export OCI_TAG=$(echo $GITHUB_REF | sed -e "s|refs/tags/||")
-          {{ "./kwctl push policy-annotated.wasm ${{ env.OCI_TARGET }}:${OCI_TAG}" }}
+
+          echo Pushing policy to OCI container registry
+          {{ "IMMUTABLE_REF=$(kwctl push -o json policy-annotated.wasm ${{ env.OCI_TARGET }}:${OCI_TAG} | jq -r .immutable_ref)" }}
+
+          echo Keyless signing of policy using cosign
+          {{" cosign sign ${IMMUTABLE_REF}" }}
```

Comment:
```suggestion
          cosign sign ${immutable_ref}"
```

Reply:
Cannot take it, it would break cargo generate